src/middleware/DefaultMiddleware.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (
       this.app.use(bodyParser.urlencoded({ extended: false }));
       this.app.use(bodyParser.json());
       this.app.use(methodOverride());
-      this.app.use(expressDevice.capture());
+      this.app.use(expressDevice.capture({ parseUserAgent: true }));
```

Comment:
This just adds the name of the device to the `request.device` object. Previously it would return `type: 'phone', name: ''`, with this change it will return `type: 'phone', name: 'iPhone'`. Might be good for logging/stats or other reasons.